sensor_msgs/Image image
darknet_ros_msgs/BoundingBox[] boxes
---
geometry_msgs/Pose pose